cmd/oci-runtime-tool/validate.go

```diff
@@ -465,17 +450,33 @@ func envValid(env string) bool {
 	return true
 }
 
-func capValid(capability string) bool {
-	for _, val := range defaultCaps {
-		if val == capability {
-			return true
-		}
-	}
+func capValid(c string, hostSpecific bool) bool {
+    cp := strings.ToUpper(c)
+
+    for _, cap := range capability.List() {
+        if cp == fmt.Sprintf("CAP_%s", strings.ToUpper(cap.String())) {
+            if hostSpecific && cap > lastCap() {
```

Comment:
I'm not sure `List` is the right logic for host-agnostic checks.  The [docs say][1]:

> List returns list of all supported capabilities

What we want here is a list of all *possible* capabilities.  But maybe I'm just misreading the docs.  Does someone have an old kernel around to test this on?

[1]: https://godoc.org/github.com/syndtr/gocapability/capability#List